packages/sources/react/src/components/structure/VtmnCard/VtmnCard.tsx

```diff
@@ -56,28 +61,31 @@ export const VtmnCard = ({
   headingLevel = 2,
   children,
   className,
+  topActions,
 }: VtmnCardProps) => {
   const Heading =
     headingLevel >= 0 && headingLevel <= 6
       ? (`h${headingLevel}` as keyof JSX.IntrinsicElements)
       : 'h2';
 
   return (
-    <div className="vtmn-card-container">
-      <div
-        className={clsx(
-          'vtmn-card',
-          `vtmn-card_variant--${variant}`,
-          className,
```

Comment:
Separating vtmn-card and variant to use the vtmn-card css. Removing container as no css is apply to it.